### ISSUE TITLE
Dashboard Token Listing: EUROe 

### ISSUE THREAD

**Issue by SFServer** [superfluid-finance/assets#142]
Discord Handle of Submitter: juicy#5959

Token Symbol: EUROe

Network: Ethereum Mainnet

Token Address: 0xd27dd3deec7eb1f1f48d9eb66f4a548c8cc04889

Network Explorer Link: https://etherscan.io/token/0xd27dd3deec7eb1f1f48d9eb66f4a548c8cc04889

Is a Super Token? true

Address of Underlying Token:  0x820802fa8a99901f52e39acd21177b0be6ee2974

SVG Logo Link: https://dev.euroe.com/img/euroe-token-symbol-svg/euroe-256x256-round.svg

Coingecko or Hardcoded Price?: Coingecko

Coingecko ID: https://www.coingecko.com/en/coins/euroe-stablecoin

Harcoded Price (if Coingecko ID is unavailable): 

Token HEX Color: #243c49

Info on Listing: EUROe is the only EU-regulated stablecoin. I'm listing it to make superfluid payments without triggering taxable events.

Please close only after token has been added to resolver!

**Comment by msoni89** [comment]
Already available on assets.

**Comment by vmichalik** [comment]
@msoni89 I removed your comment as it was incorrect, #153 was a duplicate of this issue and I already closed it
EUROe is not yet listed in the resolver on Ethereum mainnet

**Comment by vmichalik** [comment]
we do not plan to list EUROe on mainnet at this time unless there is sufficient demand to use it
